src/types.rs

```diff
@@ -585,6 +634,18 @@ impl fmt::Display for Duration {
 #[cfg_attr(feature = "fuzz", derive(Arbitrary))]
 pub struct AccountAddress(pub [u8; ACCOUNT_ADDRESS_SIZE]);
 
+#[cfg(feature = "std")]
+impl quickcheck::Arbitrary for AccountAddress {
+    fn arbitrary(g: &mut Gen) -> AccountAddress {
+        //todo should it satisfy any properties, or are random bytes okay?
```

Comment:
Random bytes are indeed OK. There are no requirements for the address in this format. 

But again, could we not just derive this?